test/test_future_and_futures.py

```diff
@@ -0,0 +1,29 @@
+import os
+import sys
+import imp
+from django.test import TestCase
+
+
+class FutureAndFuturesTestCase(TestCase):
+
+    def test_import_concurrent_py3(self):
+        import concurrent
+        if sys.version_info[0] == 3:
+            # Python 3 is supposed to import its builtin package `concurrent`
+            # instead of the one inside kolibri/dist
+            self.assertTrue('kolibri/dist' not in concurrent.__file__)
```

Comment:
Can you also check the location of the concurrent module, to ensure that it's not in the dist folder?

Reply:
That should be possible through `import concurrent; assert ('kolibri/dist' not in concurrent.__file__)`

Reply:
Yup I will change that